pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-apiserver
-    version: v1.3.0-beta.2
```

Comment:
What do you mean? I removed all version labels for consistency, since we need to remove them for deployments. Also, I don't know what it buys us, having the version in the manifest 3 times per manifest file. 2 labels + image tag seems overkill?